src/utils.jl

```diff
@@ -415,3 +415,63 @@ macro hijack_observable(name)
     end)
 
 end
+
+############################################################
+#                                                          #
+#     Splitting Of LineString / coast lines                #
+#                                                          #
+############################################################
+
+#`LineSplitting` module defines a `split` method for vectors of `LineString` objects.
+#
+#This is needed to fix e.g. coast line displays when lon_0 is not 0 but cutting polygons at lon_0+-180.
+
+module LineSplitting
+
+	import GeoMakie.LineString
+	import GeoMakie.Observable
+	import Base.split
+	
+	function regroup(tmp::Vector)
+		coastlines_custom=LineString[]
+		println(typeof(coastlines_custom))
+		for ii in 1:length(tmp)
+			push!(coastlines_custom,tmp[ii][:]...)
+		end
+		coastlines_custom
+	end
+	
+	function split(tmp::Vector,lon0=-160.0)
+		[split(a,lon0) for a in tmp]
+	end
+	
+	function split(tmp::LineString,lon0=-160.0)
```

Comment:
Same here, `lon0` should be a required argument
```suggestion
	function split(tmp::LineString, lon0::Real)
```

Reply:
Not sure. If keyword arg were used then the split function could have bypass inside if not specified maybe?